build.gradle

```diff
@@ -153,7 +153,12 @@ project(':ambry-utils') {
         compile "org.json:json:$jsonVersion"
         compile "net.sf.jopt-simple:jopt-simple:$joptSimpleVersion"
         compile "io.netty:netty-buffer:$nettyVersion"
-        testCompile "org.apache.helix:helix-core:$helixVersion"
+        testCompile ("org.apache.helix:helix-core:$helixVersion") {
+            // TODO Once new release is out, we can remove this
+            exclude group: "org.apache.helix", module: "helix-common"
+        }
+        // TODO remove zookeeper dependency once Helix open source fix is out
+        testCompile "org.apache.zookeeper:zookeeper:3.4.13"
```

Comment:
Nit, maybe could set the version number in the property file to make it consistent with other version names: `$zookeeperVersion`?

Reply:
For a permanent change we would do this.  For a temporary workaround it's okay.